### PR TITLE
Following/Followed User List

### LINKED ISSUE #95: Follower/Following List
**Is your feature request related to a problem? Please describe.**
프로필에서 Follower/Following List를 볼 수 있는 화면을 만든다.

**Describe the solution you'd like**
Back : API 작업
Front : API 통신 후 리스트 작업

### PR DESCRIPTION
**Pull request by abcthing**
Related Issue: #95 
It resolves #95 

# Major changes

- Backend

user가 팔로우하고 있는 [user]를 리턴하는 API
> GET /api/user/following

user를 팔로우하고 있는 [user]를 리턴하는 API
> GET /api/user/followed

- Frontend

FollowingList 화면 작업
FollowerList 화면 작업
ffff

# Minor changes

All test done

### Checklist
This pull request...
- [x] has preceding issues related with it (if resolves some issues, metions them with 'resolve')
- [x] doesn't include too many changes (you should focus on the specific feature in a PR)
- [x] elaborately explains what feature is added, which part is fixed or improved
- [x] has step by step instructions for collaborators, if needed
- [x] has attached screenshots, if appropriate

My codes...
- [x] include enough tests for added and changed parts
- [x] passe all the new and existing tests
- [x] include appropriate comments
- [x] don't introduce unnecessary warnings

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -59,8 +59,8 @@ function App(props) {
                             <Route path="/account_setting" exact component={AccountSetting} />
                             <Route path="/collections" exact component={CollectionList} />
                             <Route path="/collection_id=:collection_id" exact component={CollectionDetail} />
-                            <Route path="/profile_id=:profile_id/followers" exact component={FollowerList} />
-                            <Route path="/profile_id=:profile_id/followings" exact component={FollowingList} />
+                            <Route path="/followers=:profile_id" exact component={FollowerList} />
```
Comment (davin111) [inline frontend/src/App.js]: Why did you change urls? And I think if you want to change them, you should any related redirection parts of codes(for example, in Profile Detail Page).

Comment (abcthing) [reply frontend/src/App.js]: 아, follower와 following 화면 url 얘기 나눈 적이 있었나요? 저는 앞에 profile_id로 path가 한 댑스가 있는 것이 개인적으로 의아해서 followers 혹은 followings를 보는 화면에서 id만 받도록 수정한 것입니다.

Comment (davin111) [reply frontend/src/App.js]: Yes, we did. And I think urls like `/followers=id` don't seem adequate. Because the id is not that of 'followers' or 'followings'.

Comment (abcthing) [reply frontend/src/App.js]: 네네 그런 이유라면 고쳐야하겠네요! 그렇다면 /followers/profile_id=id, /followings/profile_id=id 로 하는게 어떨까요?

Comment (abcthing) [reply frontend/src/App.js]: 기존에 profile_id가 path 앞 쪽에 있던 이유가 따로 있나요?

Comment (davin111) [reply frontend/src/App.js]: Actually not, but I think most urls(not backend urls) have those rules. (grandparent/parent/child/...)

Comment (davin111) [reply frontend/src/App.js]: For example, GitHub has urls like `https://github.com/davin111?tab=followers`.

Comment (davin111) [reply frontend/src/App.js]: For example, GitHub has urls like `https://github.com/swsnu/swpp2019-team3/pulse`.

Comment (abcthing) [reply frontend/src/App.js]: 네.. 원래대로 할게요..
